docs/source/apis/product.rst

```diff
@@ -0,0 +1,402 @@
+########
+Products
+########
+  
+
+According to `Amazon’s documentation <http://docs.developer.amazonservices.com/en_US/products/Products_Overview.html>`_: 
+
+
+   The Products API section of Amazon Marketplace Web Service (Amazon MWS) helps you get information to match your products to existing product listings on Amazon Marketplace websites and to make sourcing and pricing decisions for listing those products on Amazon Marketplace websites. The Amazon MWS Products API returns product attributes, current Marketplace pricing information, and a variety of other product and listing information.
+  
+
+The Products API is available in all marketplaces and consists of 15 different operations:
+  
+   `ListMatchingProducts <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#listmatchingproducts>`_ = `list_matching_products() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L29>`_
```

Comment:
Your links are to your own repository - could you adjust them to point to the main repo (here!) please?